src/Adapter/ContainerBuilder.php

```diff
@@ -57,16 +62,80 @@ public static function getContainer($name, $isDebug)
         }
 
         $container = new SfContainerBuilder();
+
+        $parameters = require _PS_ROOT_DIR_ . '/app/config/parameters.php';
+        foreach ($parameters['parameters'] as $parameter => $value) {
+            $container->setParameter($parameter, $value);
+        }
+        $env = $isDebug ? 'dev' : 'prod';
+        $container->setParameter('kernel.bundles', []);
+        $container->setParameter('kernel.root_dir', _PS_ROOT_DIR_ . '/app/');
+        $container->setParameter('kernel.name', 'app');
+        $container->setParameter('kernel.debug', $isDebug);
+        $container->setParameter('kernel.environment', $env);
```

Comment:
what if we are in `test` environnement? (like when we launch test suites)

Reply:
This container did not manage the test environment originally, should I add a third parameter?